src/index.ts

```
@@ -0,0 +1,369 @@
+        logDate(), process.pid, newWorker.process.pid, Reflect.get(newWorker, 'state'),
+        reforkTimes.length);
+    } else {
+      log('[%s] [cfork:master:%s] don\'t fork new work (refork: %s, reforkCount: %d)',
+        logDate(), process.pid, refork, reforkTimes.length);
+    }
+  });
+
+  cluster.on('exit', function(worker, code, signal) {
+    const disableRefork = getDisableRefork(worker);
+    const log = console[disableRefork ? 'info' : 'error'];
+    const isDead = worker.isDead && worker.isDead();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)